shared/middleware/animation_library_api.rb

```diff
@@ -110,6 +110,38 @@ class AnimationLibraryApi < Sinatra::Base
     not_found
   end
 
+  #
+  # GET /api/v1/animation-library/default_spritelab/
+  #
+  # Retrieve the default sprite list from S3
+  get %r{/api/v1/animation-library/default_spritelab} do
+    result = Aws::S3::Bucket.
+      new(ANIMATION_LIBRARY_BUCKET, client: AWS::S3.create_client).
+      object("animation-manifests/manifests_levelbuilder/defaults.json").
+      get
+    content_type 'application/json'
```

Comment:
should this be hardcoded, or should it be set to `result.content_type`?

Reply:
I'm not sure what the protocol is here, but I was still experiencing the unknown content-type issue that we were seeing earlier this week when I change this to `result.content_type`.

Reply:
ah, okay -- i think it's fine to leave it hardcoded since we're explicitly requesting a JSON file here